app/src/main/java/fr/free/nrw/commons/customselector/ui/selector/CustomSelectorActivity.kt

```diff
@@ -74,12 +77,41 @@ class CustomSelectorActivity: BaseActivity(), FolderClickListener, ImageSelectLi
         }
 
         // Open folder if saved in prefs.
-        if(prefs.contains(FOLDER_ID)){
+        if(prefs.contains(FOLDER_ID)) {
             val lastOpenFolderId: Long = prefs.getLong(FOLDER_ID, 0L)
             val lastOpenFolderName: String? = prefs.getString(FOLDER_NAME, null)
             val lastItemId: Long = prefs.getLong(ITEM_ID, 0)
             lastOpenFolderName?.let { onFolderClick(lastOpenFolderId, it, lastItemId) }
         }
+
+            if(isImageGalleryEmpty())
+        {
+            val linearLayout: LinearLayout = findViewById<View>(R.id.linear_layout_noimage) as LinearLayout
+            val textView = TextView(this)
+            textView.text = "No Image Found"
+            textView.textSize = 20f
+            linearLayout.addView(textView)
+        }
+    }
+
+    /**
+     * Checks for no image in gallery.
+     */
+        private fun isImageGalleryEmpty(): Boolean {
```

Comment:
Indentation.

Reply:
what do you mean ?

Reply:
There should be only 4 spaces before `private` (same number as before `/**`).

Reply:
okay got it